source/settings/plugin/advanced/obox.rst

```diff
@@ -378,3 +378,31 @@ The weight adjustment calculation is:
   * 258 MB: ``+30``
   * 500 MB: ``+60``
   * >=1 GB: ``+120``
+
+
+.. _plugin-obox-storage-settting_obox_copy_between_users:
+
+``obox_copy_between_users``
+---------------------------------
+
+.. versionadded:: v2.3.16
+
+If set to yes this setting enables sharing mail objects in storage between
+users. If enabled it can reduce the storage usage, for example mails
+with multiple recipients will be only stored once. This does not work between
+different backends, so deduplication only works for users that live on the same
+backend.
```

Comment:
I don't think anything prevents this from working on obox side. The problem is just that copying between users is done only within LMTP session, and LMTP session only handles users in the same backend.